src/tools/wix/Compiler.cs

```diff
@@ -3451,11 +3451,6 @@ private void ParseCustomActionElement(XElement node)
                 this.core.OnMessage(WixErrors.CustomActionIllegalInnerText(sourceLineNumbers, node.Name.LocalName, innerText, "Script"));
             }
 
-            if (MsiInterop.MsidbCustomActionType64BitScript == (bits & MsiInterop.MsidbCustomActionType64BitScript) && MsiInterop.MsidbCustomActionTypeVBScript != targetBits && MsiInterop.MsidbCustomActionTypeJScript != targetBits)
-            {
-                this.core.OnMessage(WixErrors.IllegalAttributeWithoutOtherAttributes(sourceLineNumbers, node.Name.LocalName, "Win64", "Script", "VBScriptCall", "JScriptCall"));
```

Comment:
There is no way to explicitly force a 64-bit flag on a 32-bit build, so this build break is now impossible to happen.